zipa/magic.py

```diff
@@ -28,6 +28,7 @@ def __getattr__(self, name):
         return self.env[name]
 
     def _parse_name(self, name):
+        name = name.replace('___', '-')
```

Comment:
If you replace __ with - and then split on it, you'll get one element. Are you sure this is what you want?